src/plugin.video.icdrama/addon.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.icdrama" version="2.2.0" name="Icdrama" provider-name="aznhusband, dumbo0001, s_h, boleanly, mugol, General-Meow, cthlo">
+<addon id="plugin.video.icdrama" version="2.2.1+matrix.1" name="Icdrama" provider-name="aznhusband, dumbo0001, s_h, boleanly, mugol, General-Meow, cthlo">
```

Comment:
I think we should use the same version number as Leia and postfix with `+matrix.1` just like mentioned in https://forum.kodi.tv/showthread.php?tid=351745&pid=2932787#pid2932787

Reply:
Yea if we are keeping both versions then I agree with this.